frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/healthchecks.scss

```diff
@@ -0,0 +1,19 @@
+.healthchecks-scrollable {
+  overflow-y: scroll;
+  max-height: 8rem;
+}
+
+.align-troubleshoot {
+  padding-right: 0.1rem;
+}
+
+.stylize-hr {
+  padding: 0.2rem;
+  margin: 0rem;
+  border-top: 0.1rem solid grey;
+}
+
+.no-decorations {
+  outline: none;
+  text-decoration: none;
+}
```

Comment:
ceph-health-check-card {
  &__troubleshoot {
    max-height: 150px;  // put you card max size
    overflow-y: auto;  // this will takecare of scroll when content cross max-height no need > 5 check
    padding-right: var(--pf-global--spacer--md);   // md or sm your can decide
  }
  &__no-border {
    outline: none;
  }
  &__stylize-hr {
    padding: var(--pf-global--spacer--md);  // sm or md or xs
    margin: 0 var(--pf-global--spacer--sm);  // this one not sure please check
    border-top: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);   // 100 or 200
  }
}